.github/workflows/diff.yml

```diff
@@ -15,70 +15,102 @@ jobs:
   diff-reports:
     runs-on: macos-latest
```

Comment:
Yes, memory requirements - diffs are costly because two (potentially large) ontologies need to be loaded at once. macos-latest as twice as much memory as ubuntu!

Reply:
memory, uberon be big, something about macos having more memory to play with or smth, @ubyndr can correct me/explain better

Reply:
Ah yes, I seem to remember a tech-support discussion about that. Thanks for the reminder, sorry for the noise.